graphs/graph_list.py

```diff
@@ -90,65 +140,18 @@ def add_edge(self, source_vertex: int, destination_vertex: int) -> None:
                 self.adj_list[source_vertex] = [destination_vertex]
                 self.adj_list[destination_vertex] = []
 
-    def print_list(self) -> None:
+        return self
+
+    def __str__(self) -> str:
```

Comment:
OK... If `__str__()` and `__repr__()` are going to return the same values then we can delete  `__str__()` but leave the tests unmodulated.

Reply:
Done